setup.py

```diff
@@ -37,24 +39,20 @@ def read(*rnames):
         "Intended Audience :: Developers",
         "License :: OSI Approved :: Zope Public License",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
         "Natural Language :: English",
         "Operating System :: OS Independent",
         "Topic :: Software Development :: Libraries :: Python Modules",
         "Framework :: Plone",
-        "Framework :: Zope :: 2",
         "Framework :: Zope :: 3",
```

Comment:
I believe this should go away. Was Zope 3 ever Python 3-compatible?
```suggestion
```

Reply:
As Zope 3 is mostly the collection of libraries in the `zope.*` and `zope.app.*` namespaces: Yes it is Python 3 compatible, and it is not completely dead. There are some applications in the wild building purely on the Zope 3 technology stack.
So I'd like to keep this specifier as it tells that this library cannot be only used in the Zope-2-and-successors context but also without the `Zope` package.